pkgs/misc/drivers/spdk/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, numactl
+, libaio
+, cunit
+, libuuid
+, fetchFromGitHub
+, gcc
+, openssl
+, python
+}:
+
+stdenv.mkDerivation rec {
+  name = "spdk-${version}";
+  version = "v18.04-pre";
```

Comment:
Any reason for this? If possible I think we'd prefer v18.01 (and use the corresponding tag).